LIMS_IMAGE/web/laboratory/migrations/0025_alter_inventoryitem_last_ordered.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 3.2.12 on 2022-03-29 06:23
```

Comment:
you don't need to commit this change, it changes everyday based on the current date